packages/page-accounts/src/Accounts/index.spec.ts

```diff
@@ -299,6 +301,178 @@ describe('Accounts page', () => {
 
       expect(summary).toHaveTextContent(showBalance(4000 + 5000));
     });
+
+    describe('sidebar', () => {
+      const injectedAddress = '5CcZRy9WTK3NBXNxcrwK67EsVRAUsfxSQAhmSwJHtGtYwJqu';
+      const initialName = 'ANAME';
+      const newName = 'CHARLIE';
+      const defaultTag = 'Default';
+
+      let accountRows: AccountRow[];
+      let sideBar: Sidebar;
+
+      describe('changes', () => {
+        beforeEach(async () => {
+          addAccountToKeyring(injectedAddress, { isDevelopment: false, name: initialName });
+          renderAccountsForAddresses(
+            injectedAddress
+          );
+          sideBar = await openSidebarForAccountRow(0);
+        });
+
+        describe('name', () => {
+          beforeEach(async () => {
+            await sideBar.changeAccountName(newName);
+          });
+
+          it('within keyring', () => {
+            const changedAccount = keyring.getAccount(injectedAddress);
+
+            expect(changedAccount?.meta?.name).toEqual(newName);
+          });
+
+          it('within sidebar', async () => {
+            await sideBar.assertAccountName(newName);
+          });
+
+          it('within account row', async () => {
+            await accountRows[0].assertAccountName(newName);
+          });
+        });
+
+        describe('tags', () => {
+          beforeEach(async () => {
+            await sideBar.setTag(defaultTag);
+          });
+
+          it('within sidebar', async () => {
+            await waitFor(() => sideBar.tagsEqual(defaultTag));
```

Comment:
Lines like this don't test anything. `waitFor` wait for the callback not to throw, or for the promise not to reject. It does not check the boolean result from the promise.

Reply:
I removed those test as I couldn't test tags